Xamarin.Forms.Core/Slider.cs

```diff
@@ -41,6 +41,14 @@ public class Slider : View, IElementConfiguration<Slider>
 				eh(slider, new ValueChangedEventArgs((double)oldValue, (double)newValue));
 		});
 
+		public static readonly BindableProperty MinimumTrackColorProperty = BindableProperty.Create(nameof(MinimumTrackColor), typeof(Color), typeof(Slider), default(Color));
+
+		public static readonly BindableProperty MaximumTrackColorProperty = BindableProperty.Create(nameof(MaximumTrackColor), typeof(Color), typeof(Slider), default(Color));
+
+		public static readonly BindableProperty ThumbColorProperty = BindableProperty.Create(nameof(ThumbColor), typeof(Color), typeof(Slider), default(Color));
+
+		public static readonly BindableProperty ThumbImageProperty = BindableProperty.Create(nameof(ThumbImage), typeof(FileImageSource), typeof(Slider), default(FileImageSource));
```

Comment:
can't it be an `ImageSource` instead of a `FileImageSource` ?

Reply:
I used this because same was used in many renderers.